### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sarah-svg**

### CODE REVIEW

File: cart/data-cart.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { items, cart } from '../data.js';
+import { findById } from '../utils.js';
+
+const total = calculateTotal(cart);
+
+//const totalCell = document.querySelector('.total');
+document.getElementById('total').innerHTML = total;
+//totalCell.textContent = `Total: $${total}`;
+
+function calculateTotal(cartArray) {
```
Comment (breeannb) [inline cart/data-cart.js:10]: wonderful job! Just looking for testing of these functions as well :)

File: cart/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/cart.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
```
Comment (breeannb) [inline cart/index.html:16]: As Dani went over in class, adding the links to your other pages will allow it to show in Github pages.

File: cart/data-cart.js (hunk 1 above)
Comment (sarah-svg) [reply cart/data-cart.js:10]: Ok thank you I actually have part of one wrote out just not finished yet 😪   and before lunch I got it all linked together... yay!!!!!!! Thank you so much

File: cart/index.html (hunk 2 above)
Comment (sarah-svg) [reply cart/index.html:16]: Yes I tried so hard last night and couldn’t get it I wasn’t much off but was off I actually got that done before lunch.... 😊

**Pull request merged**